homeassistant/components/xiaomi_miio/remote.py

```diff
@@ -130,6 +139,11 @@ async def async_service_handler(service):
 
         device = entity.device
 
+        if service.service == SERVICE_SET_LED_ON:
+            return device.set_indicator_led(True)
```

Comment:
Please refactor and use our entity service helper instead:
https://developers.home-assistant.io/docs/dev_101_services#entity-services

Reply:
Ok :) Done